docs/gui/visualize.md

```diff
@@ -44,81 +44,62 @@ Then you can select the range of the frame in Start/End Index of left sidebar. A
 Click on the PLAY button within the plotting box to play the loaded movie.
 The number indicated on the right of PAUSE button is the frame interval in milliseconds.
 
-
 ### Customizing visualization parameters
-Select one of the display boxes by clicking inside of the box. Selected box will be highlighted by blue outline.
+Select one of the display boxes by clicking inside of the box. The selected box will be highlighted with a blue outline.
 
-Left sidebar shows the parameters available for the selected box.
+The left sidebar shows the parameters available for the selected box.
 
 <p align="center">
 <img width="600px" src="../_static/visualize/customize_param.png" alt="CustomizeParam" />
 </p>
 
+## ROI and timecourses
+In order to visualise the fluorescence time series of ROI, two visualise boxes must be created and linked.
 
-## ROI and timecourse
-There is a way to link ROI plots and fluorescence time series.
-Create One box showing ROI and another box showing fluorescence. You can link two boxes by setting `ref image` in the fluorescence box to be the ID of the ROI box. (ID of the box is on the left upper side). By clicking on the ROI of a cell, you can visualize the corresponding fluorescence time course in the fluorescence box. By turning on the `drag select` in ROI box, you can select multiple cells in the image at once.
-
-
-### Ref Image
+First create one box with the imaging data. Select the imaging data file (e.g. .tiff) from the `Select Item` drop down menu.
 
+<br>
 <p align="center">
-<img width="600px" src="../_static/visualize/ref_image.png" alt="RefImage" />
+<img width="200px" src="../_static/visualize/select_item.png" alt="Whole" />
 </p>
 
-TimeSeries plot can refer to a image plot to synthronize cell number index. Click or drag image plot so that cell number indexes are synthoronized in corresponding timeseries plot.
-
-
-### Showing ROI and time courses
-After running the ROI detection algorithms, the most often created plots are extracted cells' shape and fluorescence time series. To show the plot, prepare two plotting boxes.
+Then select the ROI (e.g. `cell_roi`). Both Suite2P and CaImAn include the process to remove the extracted ROIs that do not meet the criteria. You can visualise ROI that have been removed by selecting `non_cell_roi`.
 
 <br>
 <p align="center">
-<img width="600px" src="../_static/visualize/twobox.png" alt="Whole" />
+<img width="200px" src="../_static/visualize/select_roi.png" alt="Whole" />
 </p>
 
-In one plotting box (ex, the one with ID:0), select a background image such as meanimg from the Select Item pulldowns.
+The plotting box (#0) shows the background image and detected cells.  In OptiNiSt, a cell ID is given to all the ROIs and will be used for visualisation. Hover over an ROI to see the cell ID. 
```

Comment:
Although it is a low priority, it would be good to update the images as there seem to be some areas where the text description does not match the images.

Reply:
Images updated

Reply:
Since I can't see the result of the modification, can you please attach the commit hash and the image of the modified version?

Reply:
I'm sorry. Here's the hash: d61227b6ac4daf032371ed7f76003ad2d6425ede
<img width="732" alt="image" src="https://github.com/user-attachments/assets/591da0d3-cbec-4895-99fd-088f4fd816c4">